rockcraft.yaml

```diff
@@ -2,7 +2,7 @@ name: identity-platform-login-ui
 
 base: bare
 build-base: ubuntu:22.04
-version: '0.1'
+version: '0.6.1' # x-release-please-version
```

Comment:
This is required by the `google-release-please`. Please refer to this [doc](https://github.com/googleapis/release-please/blob/main/docs/customizing.md#updating-arbitrary-files).

Reply:
Also, we need to update the `version` in the `rockcraft.yaml` file to be the most updated version. Otherwise, the `google-release-please` action won't do any update. The `version` field will be automatically updated in the release pull request.